spec/helpers/table_helper_spec.rb

```diff
@@ -0,0 +1,82 @@
+require "rails_helper"
+
+RSpec.describe TableHelper, type: :helper do
+  describe "#sort_table_header", :sort_table_header do
+    let(:organisation) { build(:organisation) }
+    let(:header_label) { "Last Updated" }
```

Comment:
it would be good to use the same names as in the helper. If you use `heading` keeping it consistent in the test will make it easier to read.
Same applies to `header_content_attribute`